var/spack/repos/builtin/packages/dbcsr/package.py

```diff
@@ -33,8 +33,9 @@ class Dbcsr(CMakePackage, CudaPackage, ROCmPackage):
 
     depends_on('cmake@3.12:', type='build')
     depends_on('py-fypp', type='build')
-    depends_on('pkgconfig', type='build')
     depends_on('python@3.6:', type='build', when='+cuda')
+    depends_on('python@3.6:', type='build', when='+rocm')
+    depends_on('pkgconfig', type='build', when='smm=libxsmm')
```

Comment:
... since a lot of other packages pull in `pkgconfig` I stopped adding conditionals for this one at some point, but ok

Reply:
yeah... let me undo, it's hard to keep track as things change